src/xtgeo/common/xtgeo_dialog.py

```diff
@@ -577,7 +577,7 @@ def error(self, string):
 
         self._output(idx, level, string)
 
-    def critical(self, string, sysexit=True):
+    def critical(self, string, sysexit=False):
```

Comment:
Should we even allow it to `raise SystemExit`? Just remove it? And we should rather go through the locations which uses `logger.critical` and `raise` an `Exception` there instead.

Reply:
That I have already done (except perhaps for some tests), I could remove it altogether.

Reply:
I would favour removing it.